mrtt-ui/src/components/QuestionNav.js

```diff
@@ -107,7 +108,7 @@ const QuestionNav = ({ isFormSaving, isFormSaveError, onFormSave, currentSection
   const handleOnPrivacyChange = (event) => {
     setIsPrivacySaveError(false)
     setIsPrivacySaving(true)
-    if (siteFromApi) {
+    if (siteFromApi && !isFormPrivacyDisabled) {
```

Comment:
A bit redundant since the input would be disabled, but if we skip the back end part of this I feel the redundancy here is worth it as an extra 'make sure'